src/main/java/com/st/utopia/cancellationservice/controller/CancellationServiceController.java

```diff
@@ -0,0 +1,38 @@
+package com.st.utopia.cancellationservice.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.st.utopia.cancellationservice.model.Ticket;
+import com.st.utopia.cancellationservice.service.TicketService;
+
+import javassist.NotFoundException;
+
+@RestController
+@RequestMapping("/cancel")
+public class CancellationServiceController {
+	
+	@Autowired
+	private TicketService ticketservice;
+	
+	/**
+	 * Only the holder of the Ticket can cancel the reservation, or the agent if the ticket is booked from the counter.
+	 * 
+	 */
+	@PutMapping("/ticket/flight/{flightNumber}/rowNumber/{rowNumber}/seat/{seat}")

```

Comment:
As I said in my earlier review, in my experience camel-case is generally to be avoided in URLs; `/rowNumber/` should be (if not simply `/row/`, which is what I did in the booking service) either `row_number` or `row-number`.